aggregate/housing_security/rent_stable_three_mainteinance.py

```diff
@@ -0,0 +1,112 @@
+from utils.PUMA_helpers import borough_name_mapper, clean_PUMAs
+
+suffix_mapper = {
+    "_N": "",
+    "Percent MOE\n(95% CI)": "pct_moe", # don't love this. But the order does matter here. As the MOE is a partial string match
+    "MOE\n(95% CI)": "moe",
+    "CV": "cv",
+    "Percent": "pct",
+}
+
+def rent_stablized_units(
+    geography: str, write_to_internal_review=False
+    ) -> pd.DataFrame:
+
+    clean_data, denom = load_source_clean_data("units_rentstable")
+
+    indi_final = extract_geography_dataframe(clean_data, geography)
+    denom_final = extract_geography_dataframe(denom, geography, denom=True)
+    indi_final.columns = ["units_rentstable_"+ c for c in indi_final.columns]
+    denom_final.columns = ["units_occurental_" + c for c in denom_final.columns]
+    final = pd.concat([indi_final, denom_final], axis=1)
+    for code, suffix in suffix_mapper.items():
+        final.columns = [c.replace(code, suffix) for c in final.columns]
+
+    if write_to_internal_review:
+        set_internal_review_files(
+            [(final, "units_rentstable.csv", geography)],
+            "housing_security",
+        )  
+    return final
+
+def three_mainteinance_units(
+    geography: str, write_to_internal_review=False
+    ) -> pd.DataFrame:
+    clean_data, denom = load_source_clean_data("units_threemainteinance")
+
+    indi_final = extract_geography_dataframe(clean_data, geography)
+    denom_final = extract_geography_dataframe(denom, geography, denom=True)
+    indi_final.columns = ["units_threemainteinance_"+ c for c in indi_final.columns]
+    denom_final.columns = ["units_occu_" + c for c in denom_final.columns]
+    final = pd.concat([indi_final, denom_final], axis=1)
+    for code, suffix in suffix_mapper.items():
+        final.columns = [c.replace(code, suffix) for c in final.columns]
+
+    if write_to_internal_review:
+        set_internal_review_files(
+            [(final, "units_threemainteinance.csv", geography)],
+            "housing_security",
+        )  
+    return final 
+
+def load_source_clean_data(indicator: str) -> pd.DataFrame:
+    if indicator == "units_rentstable":
+        usecols = [x for x in range(10)]
+        sheetname = "2017 HVS Occupied Rental Units"
+    else:
+        usecols = [x for x in range(3)] + [x for x in range(11, 18)]
+        sheetname = "2017 HVS Occupied Units"
+
+    read_csv_arg = {
+        "filepath_or_buffer": "resources/housing_security/2017_HVS_EDDT.csv",
+        'usecols': usecols,
+        "header": 1,
+        "nrows": 63
+    }
+    read_excel_arg ={
+        "io": "resources/housing_security/2017 HVS Denominator.xlsx",
+        "sheet_name": sheetname,
+        "header": 1,
+        "nrows": 63,
+        "usecols": "A:G"
+    }
+    data = pd.read_csv(**read_csv_arg)
+    data.columns = [c.replace(".1", "") for c in data.columns]
+    data.columns = ['SBA', 'PUMA', 'CD Name', 'N', 'SE', 'MOE\n(95% CI)', 'CV', 'Percent',
```

Comment:
Not sure what this line is doing, for both indicators it seems like the columns already have these labels? A better process is to rename from the original labels instead of hard-coding and re-naming I think

Reply:
yeah this could be better rewritten it is actually a step to make sure the occupied rental units do not include the percentage MOE stuff so I would mess up the later extract step. Sort of hot fix for the issue and probably could be better written with drop?